### DISCUSSION TITLE
Pan the flow on draggable node with drag handle

### DISCUSSION THREAD

**Discussion by LouayH** [xyflow/xyflow#3754] [Q&A]
Hi there,

It seems that react flow won't allow the user to pan the flow while dragging a draggable node, which makes sense.

I thought by using [drag handle](https://reactflow.dev/examples/nodes/drag-handle) I will be able to pan the flow while dragging on the rest of the draggable node but it didn't.

![image](https://github.com/xyflow/xyflow/assets/28938792/aeeb77ad-ad5e-4795-8092-c4b9d68aa946)

This image is from react flow page about drag handle, you will notice that you can only pan the flow from outside the `DragHandleNode` which has a drag handle (the teal circle) to actually move the node.

Anyone can help me regarding pan on draggable nodes when holding far from drag handle?

**Comment by dsteinbach-ep** [comment]
You have to set `selectable` and `draggable` to `false` for your node. Then you might need to assign the green circle a `pointer-events: all` css property.
